cypress/support/step_definition/admin/administrateUsersSteps.spec.js

```diff
@@ -45,13 +45,13 @@ When("I click save changes button", () => {
 	management.clickSaveButton();
 });
 
-// Scenario: Deleting a student
+// Scenario: Deleting an user
 
-When("I click edit student button for {string}", (email) => {
-	management.clickEditStudentButton(email);
+When("I click edit {string} button for {string}", (role, email) => {
+	management.clickEditUserButton(role, email);
 });
 
-When("I click delete user button to delete user with email {string}", (email) => {
+When("I click delete user button to delete user with lastname {string}", (email) => {
```

Comment:
this needs to change from `(email)` to `(lastName)` and here as well
```
management.deleteUser(email);
```

Reply:
done